docsource/systemRequirements.rst

```diff
@@ -0,0 +1,8 @@
+System Requirements
+====================
+
```

Comment:
Should note that the system is currently imcompatible with ES 6+

Reply:
Frustratingly

Reply:
Added note.